lib/cogctl/actions/users/util.ex

```diff
@@ -8,9 +8,16 @@ defmodule Cogctl.Actions.Users.Util do
 
   def render(user_info, group_info, role_info, sort) do
     Table.format(user_info, false)
-        <> "\n\nGroups:\n" <> Table.format(group_info, sort)
-        <> "\n\nRoles:\n" <> Table.format(role_info, sort)
+        <> "\n\nGroups:\n" <> render_groups(group_info, sort)
+        <> "\n\nRoles:\n" <> render_roles(role_info, sort)
```

Comment:
Also not sure if there's an easy fix for this, but @imbriaco brought up a good point yesterday about out data from multiple lists. I wonder if we should only render groups and/or roles when a specific flag is present like `cogctl users info vanstee --groups`. Another option would be to move that stuff to something like `cogctl groups --user=vanstee`.

Not sure if that's a change we should make in this PR or not though.